README.md

```diff
@@ -145,77 +191,85 @@ Modify the stored decision with the given ID.
 ##### Parameters
 
 - `id`: The unique identifier of the stored decision
-- `response`: The updated [`response`](#response) information to store with the decision
 
-##### Returns
+##### Request Body
+
+The updated [`reply`](#reply) information to replace that which was previously associated with the decision.
+
+Since a `reply` object is used as the payload, the `allow` (allow/deny), `lifetime`, `permissions` (list of operations like `read`, `write`, etc.), and `path-scope` (file, directory, subdirectories) fields of the stored access decision can be modified.
```

Comment:
I wonder if `lifespan` is better than `lifetime`

Reply:
Yes it is. That's the Rust brain getting to me. Fixed in new commit, thanks!